pxtcompiler/emitter/plaintsc.ts

```diff
@@ -14,7 +14,7 @@ namespace ts.pxtc {
 
     export function getDiagnosticString(diagnostic: KsDiagnostic | Diagnostic): string {
         let ksDiagnostic: KsDiagnostic;
-        if ("file" in diagnostic) {
+        if ("file" in diagnostic && diagnostic.file != undefined) {
```

Comment:
why not just ``if(diagnostic.file) {`` ?

Reply:
KsDiagnostic does not have a file property, so it fails

<img width="633" alt="Screen Shot 2019-04-12 at 11 09 01 AM" src="https://user-images.githubusercontent.com/5615930/56057332-6a424300-5d13-11e9-9eb8-2bea22579cf2.png">